src/Sylius/Bundle/AdminApiBundle/Resources/config/services.xml

```diff
@@ -23,6 +23,8 @@
         <defaults public="true" />
 
         <service id="Sylius\Bundle\AdminApiBundle\Command\CreateClientCommand">
+            <argument />
```

Comment:
This passes an empty string now.

Reply:
`<argument>null</argument>` is the solution for passing null.

Reply:
Won't it just pass `null` as a string?

Reply:
Nope, I've checked it myself, writing an integration test for that would be 11/10 though :)